src/Resources/config/config.yml

```diff
@@ -0,0 +1,11 @@
+sylius_mailer:
+    emails:
+        invoice_generated:
+            subject: sylius.emails.invoice_generated.subject
+            template: "@SyliusInvoicingPlugin/Resources/views/Invoice/Email/invoiceGenerated.html.twig"
+
+knp_snappy:
+    pdf:
+        enabled: true
+        binary: %sylius_invoicing_plugin.knp_snappy.pdf.binary%
```

Comment:
To be honest, I don't think we need a parameter here, as anyone can overwrite this configuration by adding the following snippet to their config file:

```yaml
knp_snappy:
    pdf:
        binary: foobar
```

Reply:
What's more, if there are two plugins configuring this `knp_snappy`'s pdf binary and this one is the first one, then it would have no effect.